Mathlib/Algebra/Star/Basic.lean

```diff
@@ -308,15 +308,21 @@ theorem star_nat_cast [Semiring R] [StarRing R] (n : ℕ) : star (n : R) = n :=
   (congr_arg unop (map_natCast (starRingEquiv : R ≃+* Rᵐᵒᵖ) n)).trans (unop_natCast _)
 #align star_nat_cast star_nat_cast
 
---@[simp, norm_cast]
---theorem star_int_cast [Ring R] [StarRing R] (z : ℤ) : star (z : R) = z :=
---  (congr_arg unop <| map_int_cast (starRingEquiv : R ≃+* Rᵐᵒᵖ) z).trans (unop_int_cast _)
---#align star_int_cast star_int_cast
---
---@[simp, norm_cast]
---theorem star_rat_cast [DivisionRing R] [StarRing R] (r : ℚ) : star (r : R) = r :=
---  (congr_arg unop <| map_ratCast (starRingEquiv : R ≃+* Rᵐᵒᵖ) r).trans (unop_ratCast _)
---#align star_rat_cast star_rat_cast
+section
+-- Porting note: This takes too long
+set_option maxHeartbeats 0
```

Comment:
I have no clue why it needs so much time. But we should investigate this, because the proofs look pretty innocent.